src/main/java/org/elasticsearch/index/query/GeoPolygonFilterParser.java

```diff
@@ -76,8 +76,8 @@ public Filter parse(QueryParseContext parseContext) throws IOException, QueryPar
 
         List<GeoPoint> shell = Lists.newArrayList();
 
-        boolean normalizeLon = true;
-        boolean normalizeLat = true;
+        boolean normalizeLon = false;
```

Comment:
While it doesn't look like the `normalize` setting is currently documented for geo poly filter, it probably should be? Also, changing this here means the default for the setting would be different for geo poly vs geo point...that seems like it might be confusing for users?